<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proper</class>
 <widget class="QDialog" name="proper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>基本</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>301</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>文鼎PL细上海宋Uni</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>文件名:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_10">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <family>文鼎PL细上海宋Uni</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>类型：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_12">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>文鼎PL细上海宋Uni</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>大小：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_13">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <family>文鼎PL细上海宋Uni</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>位置：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_15">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <family>文鼎PL细上海宋Uni</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>最后修改时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_16">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <family>文鼎PL细上海宋Uni</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>最后访问时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_17">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>权限</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>331</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>所有者：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="fileowner">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>所在组：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="filegroup">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>权限</string>
        </property>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>57</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>所有者：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>57</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>所在组：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>51</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>其 他：</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="ownerread">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>41</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>读</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="ownerwrite">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>30</y>
           <width>41</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>写</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="ownerexec">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>30</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>执行</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="grpread">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>70</y>
           <width>41</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>读</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="grpexec">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>70</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>执行</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="grpwrite">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>70</y>
           <width>41</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>写</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="otherread">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>110</y>
           <width>41</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>读</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="otherexec">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>110</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>执行</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="otherwrite">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>110</y>
           <width>41</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>写</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
